apps/web/app/modules/Dashboard/Courses/CourseCard.tsx

```diff
@@ -16,6 +17,7 @@ type CourseCardProps = Pick<
   | "enrolled"
 > & {
   href: string;
+  completedLessonCount?: number;
```

Comment:
Can we move this inside the Pick<> type? As far as I can see, the course type has this field.

Reply:
No, because the `availableCourses` object doesn't contain the `completedLessonCount` property.